core/tests/protractor_utils/AdminPage.js

```diff
@@ -258,25 +271,21 @@ var AdminPage = function() {
   };
 
   this.getUsersAsssignedToRole = async function(role) {
-    await waitFor.visibilityOf(
-      roleDropdown, 'View role dropdown taking too long to be visible');
-    await roleDropdown.sendKeys('By Role');
+    await action.sendKeys('Role Drop Down', roleDropdown, 'By Role');
 
-    await roleValueOption.click();
-    await roleValueOption.sendKeys(role);
+    await action.click('Role Value Option', roleValueOption);
+    await action.sendKeys('Role Value Option', roleValueOption, role);
 
-    await viewRoleButton.click();
+    await action.click('View Role Button', viewRoleButton);
   };
 
   this.viewRolesbyUsername = async function(username) {
-    await waitFor.visibilityOf(
-      roleDropdown, 'View role dropdown taking too long to be visible');
-    await roleDropdown.sendKeys('By Username');
+    await action.sendKeys('Role Drop Down', roleDropdown, 'By Username');
 
-    await roleUsernameOption.click();
-    await roleUsernameOption.sendKeys(username);
+    await action.click('Role Username Option', roleUsernameOption);
```

Comment:
Do we need this click? Seems redundant because `action.sendKeys` will do this in the next line.